cyclient/scan_client.py

```diff
@@ -82,7 +131,7 @@ def get_service_name(scan_type):
     def _handle_exception(self, e: Exception):
         if isinstance(e, requests.exceptions.Timeout):
             raise CycodeError(504, "Timeout Error")
-        elif isinstance(e, requests.exceptions.HTTPError):
+        elif isinstance(e, (requests.exceptions.HTTPError, requests.exceptions.ConnectionError)):
```

Comment:
ConnectionError != HTTPError
you will get exception in `_handle_http_exception` for ConnectionError because   response and status_code not exist
add elif requests.exceptions.ConnectionError and throw CycodeException

Reply:
good point